packages/sanity/src/core/releases/tool/overview/ReleasesOverview.tsx

```diff
@@ -279,7 +279,16 @@ export function ReleasesOverview() {
   const handleOnCreateRelease = useCallback(
     (createdReleaseId: string) => {
       setIsCreateReleaseDialogOpen(false)
-      router.navigate({releaseId: createdReleaseId})
+
+      router.navigate(
+        {releaseId: createdReleaseId},
+        {
+          stickyParams: {
+            excludedPerspectives: '',
```

Comment:
Slightly clearer to use `undefined` (or `null` if types allows for it) instead of empty string?